ETrade/ETrade/OrderCreator.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ETrade
+{
+    public class OrderCreator
+    {
+        public Order Create(int id, string record)
+        {
+            var recordList = record.Split('\t').ToList();
```

Comment:
Преобразование в список лишнее. Метод `Split()` возвращает массив, элементы которого также можно использовать по индексам. А более `recordList` не используется.
```suggestion
            var recordList = record.Split('\t');
```